### PR TITLE
Fix extending a module into another

### PR DESCRIPTION
**Pull request by carl-stripe**
The test fails without the patch, and passes with the patch

r? @pt-stripe

### CODE REVIEW

File: lib/chalk-log.rb [hunk 1]
```diff
@@ -137,7 +137,7 @@ def log
   # class, unless I am a class" logic.
```
Comment (pt-stripe) [inline lib/chalk-log.rb]: should all the words with `class` in here be `class/module`?

Comment (carl-stripe) [inline lib/chalk-log.rb]: Probably; wilco

**Comment by pt-stripe** [comment]
cc @stripe/product-infra 
lgtm

**Comment by carl-stripe** [comment]
I got bored of waiting for the tests, so I'm just going to go for it

**Comment by carl-stripe** [comment]
(I've filed a ticket with Travis to see if we can exchange money for better performance)

**Pull request merged**